composer.json

```diff
@@ -23,6 +23,9 @@
     "drupal/drupal-extension" : "^4",
     "jarnaiz/behat-junit-formatter" : "^1.3.2"
   },
+  "conflict": {
+    "acquia/blt": "<=12.5.1"
```

Comment:
@mikemadison13 I think this will prevent people from upgrading this plugin prior to upgrading BLT, which would break because of the moved files